dockstore-webservice/src/main/java/io/dockstore/webservice/resources/LambdaEventResource.java

```diff
@@ -63,10 +61,14 @@ public LambdaEventResource(SessionFactory sessionFactory) {
     @Path("/{organization}")
     @Operation(operationId = "getLambdaEventsByOrganization", description = "Get all of the Lambda Events for the given GitHub organization.", security = @SecurityRequirement(name = ResourceConstants.JWT_SECURITY_DEFINITION_NAME))
     @ApiOperation(value = "See OpenApi for details")
+    @SuppressWarnings("checkstyle:parameternumber")
     public List<LambdaEvent> getLambdaEventsByOrganization(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user,
-            @ApiParam(value = "organization", required = true) @PathParam("organization") String organization,
-            @ApiParam(value = PAGINATION_OFFSET_TEXT) @QueryParam("offset") @DefaultValue("0") String offset,
-            @ApiParam(value = PAGINATION_LIMIT_TEXT, allowableValues = "range[1,100]", defaultValue = PAGINATION_LIMIT) @DefaultValue(PAGINATION_LIMIT) @QueryParam("limit") Integer limit,
+            @PathParam("organization") String organization,
+            @QueryParam("offset") @DefaultValue("0") Integer offset,
```

Comment:
Minor suggestion, even though your following a pattern we're using elsewhere... Make `offset` and `limit` be `int` instead of `Integer`, then change LambdaEventDAO.findByOrganization to take `int`, etc.

Because you have the `@DefaultValue` annotation, the parameter values will never be null. So you don't have to worry further down the line about doing null checks like [this one](https://github.com/dockstore/dockstore/pull/5730/files#diff-af8a0254394104a6d7b6c198e0bc4dad8a513664be381a1ab73c47007fdd940bR128); if it's a primitive instead of an object, you know it has a value.